src/randomInt.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * Generate a random integer between `from` and `to`.
+ *
+ * @param from - The minimum value.
+ * @param to - The maximum value.
+ * @returns The random integer.
+ * @signature
+ *   R.randomInt(from, to)
+ *   R.randomInt(to)
+ * @example
+ *   R.randomInt(1, 10) // => 5
+ *   R.randomInt(1n, 10n) // => 7n
+ *   R.randomInt(10) // => 5
+ *   R.randomInt(10n) // => 7n
+ * @dataFirst
+ * @category Number
+ */
+export function randomInt<T extends bigint | number>(
```

Comment:
Wasn't sure whether I should `purry` this function or not, what's the convention?

Reply:
i would say this function shouldn't be purried, and in fact always accept two arguments

Reply:
I wasn't sure whether to support the single argument option, it's just what lodash has

removed